.github/workflows/push.yml

```diff
@@ -0,0 +1,23 @@
+on: push
+
+name: CI / GitHub Actions
+
+jobs:
+  releaseMasterBranchOnly:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
looks like:
```
action "Release master branch only" {
  uses = "BinaryMuse/tip-of-branch@master"
  args = "master"
  secrets = ["GITHUB_TOKEN"]
}
```
got lost in translation i think!

Reply:
It's was here to prevent release. After removing Travis this moved to the line below before releasing. https://github.com/electron/electron-api-historian/pull/9/files#diff-d9e8161f95fdd9b8e476fd456f35a9a2R19

Reply:
ah ok cool!